bears/general/ImageDimensionBear.py

```diff
@@ -0,0 +1,70 @@
+import os
+import yaml
+import shutil
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.GemRequirement import GemRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+from sarge import run, Capture
+
+
+class ImageDimensionBear(GlobalBear):
+    """
+    Checks the dimension of an image us a gem.
+
+    More information is available at <github.com/Abhi2424shek/img_checker>
+    """
+
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    REQUIREMENTS = {GemRequirement('img_checker'),
+                    PipRequirement('pyyaml', '3.12')}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Image Dimension'}
+
+    @classmethod
+    def check_prerequisites(cls):
+        if shutil.which('img_checker') is None:
+            return 'img_checker is not installed.'
+        return True
+
+    def run(self,
+            image_file: str='',
+            width: int=0,
```

Comment:
what is the use of default `0`?  wont that mean every image is an error?

And what happens if the user sets this to `0`, or `-1`?
Look at other bears, and how they validate the user settings to reject bad settings.
Also if the docs are not good enough, create bugs to improve the docs.